packages/themes/core/global.css

```diff
@@ -0,0 +1,11 @@
+a[href] {
```

Comment:
Maybe it would be nice to add a small story like `Links` in the `Utilities` on storybook?